packages/ui/src/fleets/map/Map.tsx

```
@@ -0,0 +1,122 @@
+"use client";
+
+import { useMemo, useState, useCallback, useRef } from "react";
+import DeckGL, { FlyToInterpolator, type MapViewState, type PickingInfo } from "deck.gl";
+import Map from "react-map-gl/mapbox";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)